bothan-coinbase/src/worker/types.rs

```diff
@@ -0,0 +1,5 @@
+use tokio::time::Duration;
+
+pub const DEFAULT_CHANNEL_SIZE: usize = 1000;
+pub const DEFAULT_TIMEOUT: Duration = Duration::from_secs(720);
```

Comment:
I understand that all sources are using this number. However, may I know how it's calculated?

Reply:
@warittornc

Reply:
I also noticed that this default value is never overrided by the custom value, is it intentional? if it is should it be renamed? @warittornc

Reply:
custom value should overridden as needed (from deserialize from config in the case of the api), as for the values they're currently just arbitrary, @colmazia can change them on any solid reasoning if need be

Reply:
I think we should not let custom value since it is used before worker upgrade (in code) and it is the value depend on the source not the user.
So I decided to just change the variable name to SOURCENAME_TIMEOUT

Reply:
fixed